cli/organization.go

```diff
@@ -0,0 +1,86 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/coder/coder/v2/cli/clibase"
+	"github.com/coder/coder/v2/cli/cliui"
+	"github.com/coder/coder/v2/codersdk"
+)
+
+func (r *RootCmd) organizations() *clibase.Cmd {
+	cmd := &clibase.Cmd{
+		Annotations: workspaceCommand,
+		Use:         "organizations { current }",
+		Short:       "Organization related commands",
+		Aliases:     []string{"organization", "org", "orgs"},
+		Hidden:      true, // Hidden until these commands are complete.
```

Comment:
I marked this as hidden, I think we can defer the actual command name discussion and get the code in. Don't want to bikeshed and prevent code progress.